### PR TITLE
[Automation] Add ability to reset DB after deploy

### PR DESCRIPTION
**Pull request by yanburman**

### CODE REVIEW

File: automation/patch_igz/patch_env_template.yml [hunk 1]
```diff
@@ -33,3 +33,6 @@ REGISTRY_USERNAME:
 
 # Optional - will attempt docker login if defined
 REGISTRY_PASSWORD:
+
+# Optional - only required if asking to reset DB
+DB_USER:
```
Comment (theSaarco) [inline automation/patch_igz/patch_env_template.yml:38]: Why do you need a user? We always deploy MLRun DB with the `root` user, and I think it's the only valid user (we don't create other users for the DB). Why not always use `root`?

File: automation/patch_igz/patch_remote.py [hunk 2]
```diff
@@ -274,6 +281,121 @@ def _wait_deployment_ready(self):
...
+        logger.info("Scaling up mlrun-api-worker")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "scale",
+                "deploy",
+                "mlrun-api-worker",
+                "--replicas=1",
```
Comment (theSaarco) [inline automation/patch_igz/patch_remote.py:388]: What if there were more than 1 workers? What if there were no workers (only chief is deployed)? 
Should check how many replicas were before, and return to that situation. Alternatively - let the user enter this as a config to the script (could also be used as a different command just to scale-up/down).

**Review by theSaarco — COMMENTED** [review]
Very nice. I do have some suggestions though :-)

**Review by theSaarco — APPROVED** [review]
Cool.

**Review by liranbg — APPROVED** [review]

**Pull request merged**